### PR TITLE
Fix some grammar errors and redundant words

### PR DESCRIPTION
**Pull request by italovinicius18**
Co-authored-by: IanFPFerreira <ianfillipe@gmail.com>
Co-authored-by: guilhermemoraisr <guilhermemoraisr@gmail.com>

**Proposed changes**:
- There were words and spaces that were duplicated and some wrong syntax that was corrected

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [X] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by italovinicius18** [comment]
@m-vdb I continued the work on the errors related to this issue #10954 and reviewed the docs again for errors, so I am making this PR, because some errors passed me by

File: docs/docs/action-server/actions.mdx [hunk 1]
```diff
@@ -229,7 +229,7 @@ events, the order won't matter, but with other event types it can.
 
 ### `responses`
 
-A response can be of any of the response types described in the
```
Comment (m-vdb) [inline docs/docs/action-server/actions.mdx:232]: I think this form is correct

File: docs/docs/action-server/knowledge-base-actions.mdx [hunk 2]
```diff
@@ -170,7 +170,7 @@ you need to specify and annotate the following entities in your training example
   object type listed as a key in the knowledge base.
 
 * `mention`: If the user refers to an object via “the first one”, “that one”, or “it”, you should mark those terms
-  as `mention`. We also use synonyms to map some of the mentions to symbols. You can learn about that
```
Comment (m-vdb) [inline docs/docs/action-server/knowledge-base-actions.mdx:173]: I think this form is correct

**Review by m-vdb — CHANGES_REQUESTED** [review]
Thanks a lot @PedroLucasCM for fixing the docs 🙌🏻 I think there were 2 mistakes, can you address them?

**Comment by PedroLucasCM** [comment]
> Thanks a lot @PedroLucasCM for fixing the docs 🙌🏻 I think there were 2 mistakes, can you address them?

Hey @m-vdb I think you mistook me for @italovinicius18 :sweat_smile: we're on the same team though and already warned him

**Review by m-vdb — APPROVED** [review]

**Comment by m-vdb** [comment]
Thanks a lot @italovinicius18 🙏🏻

**Pull request merged**